FireRender.Maya.Src/FireRenderBlendMaterial.cpp

```diff
@@ -96,11 +97,12 @@ MStatus FireMaya::BlendMaterial::compute(const MPlug& plug, MDataBlock& block)
 		MFloatVector& topColor = block.inputValue(Attribute::inputA).asFloatVector();
 		MFloatVector& baseColor = block.inputValue(Attribute::inputB).asFloatVector();
 		float weight = block.inputValue(Attribute::weight).asFloat();
-
+		
 		// set ouput color attribute
 		MDataHandle outColorHandle = block.outputValue(Attribute::output);
 		MFloatVector& outColor = outColorHandle.asFloatVector();
 		outColor = topColor * weight + baseColor * (1.0f - weight);
+		outColor = topColor;
```

Comment:
Coukl we avoid assignment two times in a row ?

Reply:
fixed